src/status_im2/contexts/wallet/edit_account/view.cljs

```diff
@@ -0,0 +1,81 @@
+(ns status-im2.contexts.wallet.edit-account.view
+  (:require [quo2.core :as quo]
+            [quo2.theme :as quo.theme]
+            [react-native.core :as rn]
+            [react-native.safe-area :as safe-area]
+            [reagent.core :as reagent]
+            [status-im2.contexts.wallet.edit-account.style :as style]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(defn- view-internal
+  []
+  (let [{:keys [top]}   (safe-area/get-insets)
```

Comment:
hmm, this all seems a bit too much re-write for me. Why are we not reusing some parts from the create-account screen? at the very least we should make lots of components as we can re-use the screen pretty much. 
In the case of edit screen we just load up different defaults really 🤔

Reply:
wdyt?

Reply:
yeah I agree with your points here :). That sounds good, let's go for whatever makes things the easiest to maintain consistency between the two pages while not being a pain to adjust one or the other.

So yeah a full page component approach will be too much like you said, but breaking these down (where beneficial) to styled quo components might help here. Such as the example you mention here ^ ^.

Maybe we can start with just that anyway and then see in time if it's better to integrate these two pages more (or not)

Reply:
@J-Son89 - I have updated the PR with the screen base/template for the Create or Edit account. Please do a review of it. I haven't updated the create account screen as there is active PR for `status-go` integration. I will update it upon merging to prevent conflicts 👍.